releases/development/master/extra/barclamp_install.rb

```diff
@@ -109,15 +120,28 @@ def rm_tmpdir(tmpdir)
     puts "Barclamp at #{bc} has no name, skipping"
     next
   end
+  if barclamp["barclamp"] and barclamp["barclamp"]["version"] and \
+    barclamp["barclamp"]["version"].to_i 
```

Comment:
Its there to be consistent with extracting 'name' key from the crowbar.yml. Should I change this to version = `barclamp["barclamp"]["version"].to_i rescue nil` or similar?